ArcadiaParties.Data.Abstractions/Repositories/IUserRepository.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ArcadiaParties.Data.Abstractions.DTOs;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.Data.Abstractions.Repositories
+{
+    public interface IUserRepository
+    {
+        Task<UserDTO> GetUser(string identity);
+        Task<List<UserDTO>> GetUsers();
```

Comment:
You should return immutable collection like `IReadOnlyCollection` or `IEnumerable` instead of concrete `List`.

Reply:
Collection type changed